project-proposals/stage-2/stage-2-questionnaire.md

```diff
@@ -0,0 +1,60 @@
+# Stage 2: Incubation
+
+*As a project intending to move to stage 2 of the OpenRail incubation process, please fill out this questionnaire and send it to the Technical Committee. See the [incubation process](../../incubation-process.md) for details about criteria and requirements for new projects and how it works to get new projects into the association.*
+
+*Copy this template and fill in your answers to the questions in the sections below.*
+
+## Health metrics
+
+*How to assess project health depends on the context of the concrete project. Please answer the questions below and add any additional explanations or data you see as useful to show that the project has a healthy community and development activity.*
+
+### How many committers does the project have?
```

Comment:
We could specify that a link to the pulse page is sufficient.

Reply:
I wonder if the question in this form even makes sense. The total number of committers doesn't really say anything about the current activity of the project or how many people do significant contributions. Maybe we need a better definition of what an "active contributor" is and use that as the main criterion. This could then be merged with the "How active is the project?" question.

Reply:
From this week's TC meeting:

We look for active contributors in the following sense:
* People who were active within the last twelve months
* People who did more than x commits (let's go with x=5 for now)

We document this in the questionnaire. When submitting the questionnaire projects only have to submit the link to the GitHub Contributors page.